io/_test_common.ts

```diff
@@ -18,11 +18,14 @@ export const bufsizes: number[] = [
 
 export class BinaryReader implements Reader {
   index = 0;
+  #bytes:Uint8Array;
 
-  constructor(private bytes: Uint8Array = new Uint8Array(0)) {}
+  constructor(bytes: Uint8Array = new Uint8Array(0)) {
```

Comment:
Nit: the type can be inferred from the variable's default value. Type annotation is not needed.
```suggestion
  constructor(bytes = new Uint8Array(0)) {
```